cluster-topology.md

```diff
@@ -0,0 +1,151 @@
+# Use Cluster API to change the Cluster Topology
+
+[In the previous walk-through](./your-first-cluster.md) we demonstrated how to create a cluster using a standard declarative configuration. Now we will show how to leverage this declarative configuration to update cluster topology with ease.
+
+## What is Cluster Topology?
+
+We define a cluster topology as the set of configurations that describe your cluster: a few examples are the number of control plane and worker nodes; the type of Machine hardware that underlies various nodes; regional distribution across nodes or node pools. You may also see this described as "cluster shape" elsewhere.
+
+## Scale out control plane nodes
+
+A common Kubernetes cluster maintenance activity is scaling out (or in) the number of control plane nodes in response to cluster activity. Because Cluster API configuration interfaces are in fact Kubernetes resources, there are a lot of ways to do this. We'll demonstrate using a variety of methods.
+
+### Scale from 1 to 3 control plane nodes via kubectl apply
+
+Because we are leveraging the Kubernetes declarative model, we can simply refer to a desired configuration specification and rely upon Cluster API to evaluate what's different between the two.
+
+Assuming that your `docker-cluster-one` is in its original configuration, you should have one control plane node:
+
+```sh
+$ kubectl --kubeconfig cluster-one.kubeconfig get nodes
```

Comment:
Nit: how about breaking this into 2 sections - one for the command and one for the output. If we have a dedication one for the command then people will be able to use the "copy to clipboard button" in github and quickly copy and past the command the run the thing. 

Right now it copied the input and the output. People wont be able to use it unless they edit it locally. Would be nice to just copy paste and run.

Same comment at other places.

Reply:
Done